pkgs/by-name/co/committed/package.nix

```diff
@@ -0,0 +1,22 @@
+{ lib, fetchFromGitHub, rustPlatform }:
+rustPlatform.buildRustPackage rec {
+  pname = "committed";
+  version = "v1.0.20";
+
+  src = fetchFromGitHub {
+    owner = "crate-ci";
+    repo = pname;
+    rev = "e0a4ba358ac0e6292e750f61b74f77a347eb10ad";
+    hash = "sha256-HqZYxV2YjnK7Q3A7B6yVFXME0oc3DZ4RfMkDGa2IQxA=";
+  };
+  cargoSha256 = "sha256-aqQBkrYI3GGz3qPX6vnoXhEa+BGDbbfZIhmwcQA9qJc=";
+
+  meta = with lib; {
+    homepage = "https://github.com/crate-ci/committed";
+    description = " Nitpicking commit history since beabf39 ";
+    mainProgram = "committed";
+    license = with licenses; [ mit asl20 ];
+    maintainers = with maintainers; [ nydragon ];
```

Comment:
```suggestion
  meta = {
    homepage = "https://github.com/crate-ci/committed";
    description = " Nitpicking commit history since beabf39 ";
    mainProgram = "committed";
    license = with lib.licenses; [
      mit
      asl20
    ];
    maintainers = with lib.maintainers; [ nydragon ];
```